Core/Graphics/Source/FrameBuffer.cpp

```diff
@@ -136,6 +132,21 @@ namespace Babylon
         encoder.setStencil(m_hasStencil ? stencilState : 0);
     }
 
+    void FrameBuffer::Dispose()
```

Comment:
Does anything other than the destructor call this? I'm a bit unclear when this should be called by anything other than the destructor.